Library/src/main/java/com/wunderlist/slidinglayer/SlidingLayer.java

```diff
@@ -864,10 +865,14 @@ protected boolean canScroll(View v, boolean checkV, int dx, int dy, int x, int y
                 }
             }
         }
-
-        return checkV && (
+        boolean temp = checkV && (
                 (allowedDirection() == HORIZONTAL && ViewCompat.canScrollHorizontally(v, -dx) ||
-                        allowedDirection() == VERTICAL && ViewCompat.canScrollVertically(v, -dy)));
+                        allowedDirection() == VERTICAL && v.canScrollVertically( -dy)));
```

Comment:
There should be a benefit on using the ViewCompat library. That will take away some of the headaches of dealing with the two different API libraries. Unless this is deprecated based on the minSdkVers. Is that the case?

Reply:
ViewCompat is deprecated in our project because our minSdkVers. is 18.

Reply:
I see. And doesn't the `ViewCompat` scroll check work properly? The minimum sdk supported is something that there is no motivation to change at the moment, hence this change is something that will presumably have to stay at your fork.